tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -287,16 +396,36 @@ def run(ceph_cluster, **kw):
             + "operation : Deep-scrub\n"
             + "Expectation : No repairs to be made"
         )
-        auto_repair_param_value = no_of_objects + 1
+        if not check_for_scrub(rados_obj, wait_time):
+            log.error(
+                "Test_scenario5: The scrub operations are still in progress.Not executing the further tests"
+            )
+            return 1
+        no_of_inconsistent_objects = get_pg_inconsistent_object_count(rados_obj, pg_id)
+        auto_repair_param_value = no_of_inconsistent_objects + 1
         mon_obj.remove_config(section="osd", name="osd_scrub_auto_repair")
```

Comment:
Explicitly set the osd_scrub_auto_repair to False. makes workflow more clear.

Reply:
Modified the code.